mmv1/third_party/terraform/resources/resource_container_cluster.go.erb

```diff
@@ -457,33 +457,125 @@ func resourceContainerCluster() *schema.Resource {
 										Default:     "default",
 										Description: `The Google Cloud Platform Service Account to be used by the node VMs.`,
 									},
+
 									"image_type": {
 										Type:        schema.TypeString,
 										Optional:    true,
 										Default:     "COS_CONTAINERD",
 										Description: `The default image type used by NAP once a new node pool is being created.`,
 										ValidateFunc: validation.StringInSlice([]string{"COS_CONTAINERD", "COS", "UBUNTU_CONTAINERD", "UBUNTU"}, false),
 									},
-									<% unless version == 'ga' -%>
+
 									"min_cpu_platform": {
 										Type:             schema.TypeString,
 										Optional:         true,
 										DiffSuppressFunc: emptyOrDefaultStringSuppress("automatic"),
 										Description:      `Minimum CPU platform to be used by this instance. The instance may be scheduled on the specified or newer CPU platform. Applicable values are the friendly names of CPU platforms, such as Intel Haswell.`,
 									},
-									<% end -%>
+									"disk_size_gb": {
+										Type:         schema.TypeInt,
+										Optional:     true,
+										Default:      100,
+										ValidateFunc: validation.IntAtLeast(10),
+										Description:  `Size of the disk attached to each node, specified in GB. The smallest allowed disk size is 10GB. If unspecified, the default disk size is 100GB.`,
+									},
+									"disk_type": {
+										Type:         schema.TypeString,
+										Optional:     true,
+										Default:	  "pd-standard",
+										ValidateFunc: validation.StringInSlice([]string{"pd-standard", "pd-balanced", "pd-ssd"}, false),
+										Description:  `Type of the disk attached to each node. If unspecified, the default disk type is 'pd-standard'`,
+									},
+									"boot_disk_kms_key": {
+										Type:        schema.TypeString,
+										Optional:    true,
+										Computed:    true,
+										Description: `The Customer Managed Encryption Key used to encrypt the boot disk attached to each node in the node pool. This should be of the form projects/[KEY_PROJECT_ID]/locations/[LOCATION]/keyRings/[RING_NAME]/cryptoKeys/[KEY_NAME]. For more information about protecting resources with Cloud KMS Keys please see: https://cloud.google.com/compute/docs/disks/customer-managed-encryption.`,
+									},
+									"image_type": {
```

Comment:
This looks to be the offending line

Reply:
@dysosmus can you please check?

Reply:
Hey I will check, sorry for the slow reaction time :).